### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by olomouc** [danielcuthbert/thescum#4]
Daniel, this is a great initiative but people without strong technical skills are not going to get this to work. Youve just provided a list of ip addresses and a list of domain names and a link to editing a hosts file. So if i had no idea I'd take the lists and add them to the hosts file. Whats that going to do?

**Comment by danielcuthbert** [comment]
I hear you, so can you donate time to write a guide in simple english?

**Comment by olomouc** [comment]
Yeah sure, Ill do it in the next day or two. I'm not an expert but I would just add loopback 127.0.0.1 on all of the domain names, is that correct? Not sure what to do with the IP address files, nslookup and the same...

**Comment by ghost** [comment]
It may be worth noting you can just add the *_web files as blocklists into Gravity on [Pi Hole](https://pi-hole.net/). That might be a more useful route for some.

**Comment by danielcuthbert** [comment]
> It may be worth noting you can just add the *_web files as blocklists into Gravity on [Pi Hole](https://pi-hole.net/). That might be a more useful route for some.

Would you have any spare cycles to write this up and do some screenshots?

**Comment by danielcuthbert** [comment]
> Yeah sure, Ill do it in the next day or two. I'm not an expert but I would just add loopback 127.0.0.1 on all of the domain names, is that correct? Not sure what to do with the IP address files, nslookup and the same...

That's right. I'll try and do an example tonight

**Comment by olomouc** [comment]
I made a start on an intro, so that people dont feel like theyve entered the dark web!

When you browse a website, you are initiating a sequence that through a series of name servers locates the ip address of the location where the website physically exists. For example, 

www.bbc.co.uk -> name server -> 151.101.64.81 -> ‘here be pages’ -> browser

The ‘hosts’ file on your operating system is a personal version of this that only affects you.
The ‘Hosts’ file is used to map ip addresses to hostnames. It is most commonly used in conjunction with the system loopback ip -  127.0.0.1. If you use this ip address in conjunction with a domain name, your computer will first check this file and if it doesn’t exist it will go looking on the internet. If it does exist, then it will decide that the domain name you are trying to browse is associated with the ip address in the file and go no further. In this example, 

127.0.0.1     www.thesun.co.uk

You will get ‘unable to connect’, or similar according to your browser, as your computer is trying to find a website in your system that has no pages. 

The purpose of adding these entries to your host file is to stop you inadvertently clicking through to hate spreading websites. Clicks = $, $ = more hate. Don’t feed the machine.

**Comment by danielcuthbert** [comment]
This is great stuff, I've added an updated README, so when you feel you are ready to add it, just give me a pull request and ill merge

**Comment by Dunsford** [comment]
I'm fairly tech but couldn't get the editing Hosts to work using the instructions - it couldn't find the file to save at the end. 
These instructions worked better for me:
https://www.imore.com/how-edit-your-macs-hosts-file-and-why-you-would-want#how-to-edit-the-hosts-file

Proof instructions above (reminder to self - RTFM!)

**Comment by Dunsford** [comment]
PS Useful instructions if you do screw up your Hosts!

https://www.imore.com/how-edit-your-macs-hosts-file-and-why-you-would-want#how-to-reset-the-hosts-file
